pandas/tests/series/test_period.py

```diff
@@ -113,3 +113,13 @@ def test_to_period(self, input_vals):
         expected = Series([input_vals], dtype="Period[D]")
         result = Series([input_vals], dtype="datetime64[ns]").dt.to_period("D")
         tm.assert_series_equal(result, expected)
+
+    @pytest.mark.parametrize("comparison_data", ["a", False, 1, 1.0, None])
+    def test_comparison_operations(self, comparison_data):
+        # GH 28980
+        expected = Series([False, False])
+        result = (
+            Series([pd.Period("2019"), pd.Period("2020")], dtype="period[A-DEC]")
```

Comment:
can you separate this into two lines like

```
ser = pd.Series(...)
result = ser == comparison_data
```

Reply:
Sure, it is done. I've push an update.